apache/superset
# chore: Show datasets when search input is empty

### SUMMARY
Shows data in the table when search input is empty. Related to [this comment ](https://github.com/apache/superset/pull/12390#issuecomment-757353286).

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Before:
![BEFOREAge-distribution-of-respondents (1)](https://user-images.githubusercontent.com/60598000/104110091-9322ae00-52d4-11eb-97b3-fe19e3059022.gif)


After:

https://user-images.githubusercontent.com/60598000/104195754-50bbb700-5423-11eb-861e-f6ba43ea34a5.mp4


### TEST PLAN
1. Open a chart
2. Change the dataset
3. Search for a dataset
4. Delete the search string from the input
5. Make sure the initial data is fetched

### ADDITIONAL INFORMATION
